terraform-test/action.yaml

```diff
@@ -0,0 +1,48 @@
+name: Terraform TEST
+description: "terraform tool common flow action to init, test terraform code"
+author: Das Meta
+branding:
+  icon: globe
+  color: purple
+inputs:
+  fetch-depth: 
+    description: "Number of commits to fetch. 0 indicates all history for all branches and tags."
+    required: false
+    default: 0
+  aws-region: 
+    description: "AWS Region, e.g. us-east-2"
+    required: false
+    default: eu-central-1
+  aws-access-key-id: 
+    description: "AWS Access Key ID. This input is required if running in the GitHub hosted environment."
+    required: false 
+  aws-secret-access-key:
+    description: "AWS Secret Access Key. This input is required if running in the GitHub hosted environment."
+    required: false
+runs:
+  using: "composite"
+  steps:
+    - name: Check out code
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: ${{ inputs.fetch-depth }}
+
+    - name: Configure AWS Credentials
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
+        aws-region: ${{ inputs.aws-region }}
+        aws-access-key-id: ${{ inputs.aws-access-key-id }}
+        aws-secret-access-key: ${{ inputs.aws-secret-access-key }}
+
+    - name: Setup Terraform
+      uses: hashicorp/setup-terraform@v2
+      with:
+        terraform_version: 1.2.1
+
+    - name: Run Terraform Test
+      run: |
+        pwd
+        echo $PWD
+        cd modules/dashboard
```

Comment:
this seems can be uses only for repo which have modules/dashboard folder,
can we pass the path to repo as variable? or better if the code will get lis of folders or even better if the module will list all sub-modules and root module and run `terraform test` on them?